05-onchain.md

```diff
@@ -143,12 +143,16 @@ A node:
       sufficient fee:
         - SHOULD use this fee to perform a *mutual close*.
       - otherwise:
-        - MUST use the *last commitment transaction*, for which it has a
+      	- if the node knows or assumes its channel state is outdated it
+      	  -  MUST NOT broadcast its *last commitment transaction*. 
+      	  -  SHOULD send an `error`.
```

Comment:
Seems related to #932. I think this is a weird place to add this requirement as BOLT 2 is where we define the _p2p_ interaction, while BOLT 5 so far has been reserved primarily for _on chain_ interaction. In other words, this seems to sort of "breach" layers. 

This was discussed during the latest dev call, with some inclination to sort of merge this and #932, remove this line (as it's in #932) and keep the rest of the diff.

Reply:
@lightning-developer I've prepared a commit on top of your branch that does this: https://github.com/t-bast/bolts/commit/ae2d1e9f7128adb19ca2e9d6043505467eb19af3

Can you cherry-pick it and update this PR? We can then include it and simply close #932.

Reply:
Done as requested.